docs/source/developer_guide/guides/3_simple_cpp_stage.md

```diff
@@ -28,9 +28,9 @@ CppConfig.set_should_use_cpp(False)
 
 If a stage does not have a C++ implementation, Morpheus will fall back to the Python implementation without any additional configuration and operate in a hybrid execution mode.
 
-In addition to C++ accelerated stage implementations, Morpheus also provides a C++ implementation for message primitives. When C++ execution is enabled, constructing one of the Python message classes defined in the `morpheus.pipeline.messages` module will return a Python object with bindings to the underlying C++ implementation.
+In addition to C++ accelerated stage implementations, Morpheus also provides a C++ implementation for message primitives. When C++ execution is enabled, constructing one of the Python message classes defined in the `morpheus.messages` package will return a Python object with bindings to the underlying C++ implementation.
```

Comment:
morpheus._lib.messages, and its definitely a module, not a package. Morpheus itself is a package, but all its pybind/cython sub modules aren't.